cv-data-controller/src/main/java/com/trihydro/cvdatacontroller/controller/CdotUpstreamPathController.java

```diff
@@ -0,0 +1,385 @@
+package com.trihydro.cvdatacontroller.controller;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.trihydro.library.helpers.CdotGisConnector;
+import com.trihydro.library.model.Milepost;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.web.client.RestClientException;
+
+@CrossOrigin
+@RestController
+@RequestMapping("cdot-upstream-path")
+public class CdotUpstreamPathController extends BaseController {
+  private final CdotGisConnector cdotGisService;
+
+  private final Logger logger = LoggerFactory.getLogger(CdotUpstreamPathController.class);
+
+  @Autowired
```

Comment:
We should be able to remove this autowired since there's a single constructor https://docs.spring.io/spring-framework/reference/core/beans/annotation-config/autowired.html

Reply:
Thanks for the link! This has been removed.